### PR TITLE
nitrogen: add page

### PR DESCRIPTION
**Pull request by Dale-Muccignat**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):1.6.1-4**

### CODE REVIEW

File: pages/linux/nitrogen.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# nitrogen
+
+> Nitrogen is a fast and lightweight desktop background browser and setter for X Window.
+> More information: <https://github.com/l3ib/nitrogen/>.
+
+- View and set the desired wallpaper/s from a specific directory:
+
+`nitrogen {{/path/to/directory/}}`
```
Comment (einverne) [inline pages/linux/nitrogen.md:8]:
```suggestion
`nitrogen {{path/to/directory/}}`
```

We do not use the leading `/` here.

File: pages/linux/nitrogen.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- Set the wallpaper with automatic size settings:
+
+`nitrogen --set-auto {{/path/to/file}}`
```
Comment (einverne) [inline pages/linux/nitrogen.md:12]:
```suggestion
`nitrogen --set-auto {{path/to/file}}`
```

**Review by einverne — CHANGES_REQUESTED** [review]
hi @Dale-Muccignat thank you for the new page. I've just added a small suggestion.

**Review by einverne — APPROVED** [review]
Cool, thanks.

File: pages/linux/nitrogen.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# nitrogen
+
+> Nitrogen is a fast and lightweight desktop background browser and setter for X Window.
+> More information: <https://github.com/l3ib/nitrogen/>.
```
Comment (navarroaxel) [inline pages/linux/nitrogen.md:4]:
```suggestion
> More information: <https://github.com/l3ib/nitrogen>.
```

File: pages/linux/nitrogen.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- View and set the desired wallpaper/s from a specific directory:
+
+`nitrogen {{path/to/directory/}}`
```
Comment (navarroaxel) [inline pages/linux/nitrogen.md:8]:
```suggestion
`nitrogen {{path/to/directory}}`
```

File: pages/linux/nitrogen.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# nitrogen
+
+> Nitrogen is a fast and lightweight desktop background browser and setter for X Window.
+> More information: <https://github.com/l3ib/nitrogen/>.
+
+- View and set the desired wallpaper/s from a specific directory:
```
Comment (navarroaxel) [inline pages/linux/nitrogen.md:6]: The `/` only adds noise
```suggestion
- View and set the wallpapers from a specific directory:
```

File: pages/linux/nitrogen.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+`nitrogen {{path/to/directory/}}`
+
+- Set the wallpaper with automatic size settings:
+
+`nitrogen --set-auto {{path/to/file}}`
+
+- Restore the previous wallpaper:
+
+`nitrogen --restore &`
```
Comment (navarroaxel) [inline pages/linux/nitrogen.md:16]: The `&` is to send the command to the background? it's not part of the app params itself.
```suggestion
`nitrogen --restore`
```

File: pages/linux/nitrogen.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# nitrogen
+
+> Nitrogen is a fast and lightweight desktop background browser and setter for X Window.
```
Comment (navarroaxel) [inline pages/linux/nitrogen.md:3]:
```suggestion
> Desktop background browser and setter for X Window.
```

**Comment by Dale-Muccignat** [comment]
Thank you for your feedback and reviews thus far. They are all good points.

**Review by sbrl — APPROVED** [review]
Thanks, @Dale-Muccignat!

**Pull request merged**